File: semestr-8/Jezyk C#/alexander/Csharp/Archiwum/2021s/Id06TC1/Command/Command/Sportsmen.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Command
{
    public class Sportsmen
    {
        private string name;
        public Sportsmen(string name) { this.name=name; }
        public String Run() { return "Running"; }
        public String Jump() { return "Jumping"; }
        public String Swim() { return "Swimming"; }
        public String execute(SportsmenCommand cmd) { return name + ": " + cmd.execute(this); }
    }
}

File: semestr-8/Programowanie obiektowe w Java/alexander/POJ/2023s/IZ08IO1/StrategyProject/src/Sportsmen.java
public class Sportsmen
{
    private String name;
    private String running;
    public Sportsmen(String name,String running) 
    {
        this.name=name; 
        this.running=running;
    }
    public String Run() { return "Running"; }
    public String Jump() { return "Jumping"; }
    public String Swim() { return "Swimming"; }
}
